apps/api/src/domain/models/audio-item/entities/video.entity.ts

```diff
@@ -33,6 +33,9 @@ import {
 } from './transcribable.mixin';
 import { Transcript } from './transcript.entity';
 
+export const isVideoMimeType = (mimeType: MIMEType): boolean =>
+    [MIMEType.mp4, MIMEType.videoOgg, MIMEType.videoWebm].includes(mimeType);
```

Comment:
Is video/quicktime part of this list for MOV?

Reply:
Not yet. We can add it on a future PR. We should also add test files for each format \ extension to our `**ingest-media-item**` test case.